source/Marquee.js

```diff
@@ -717,6 +718,19 @@
 			};
 		}),
 
+		_marquee_setTextOverflow: function () {
+			var node, rect;
+			node = this.$.marqueeText ? this.$.marqueeText.hasNode() : this.hasNode();
```

Comment:
It is hard to use the _marquee_calcDistance() function here, as the only similarity is in getting the node and rect values but the logic is different and the _marquee_calcDistance() code will unnecessarily create this._marquee_distance which will cause other issues